### DISCUSSION TITLE
0.39 TOGGLE_EVENTS

### DISCUSSION THREAD

**Discussion by phebus13** [nguyenquyhy/Flight-Tracker-StreamDeck#310] [Q&A] [answered]
I noticed the TOGGLE_EVENTS page was removed from the user guide in this update. Is there a list of all events still?

**Comment by nguyenquyhy** [comment, marked as answer]
It is now moved to https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/FlightStreamDeck.Logics/KnownEvents.cs and is simply a list of suggestions. From v0.39 you can freely use any events outside of that list. 

In the next version, the list will be used for autocomplete to help users that are not too familiar with SimConnect events.

I'll update the User Guide shortly on this,

**Comment by phebus13** [comment]
Perfect, thank you!

**Reply by JoergM-D** [reply]
Where can I find the mobiflight keys/commands?

**Reply by rmroc451** [reply]
@JoergM-D ~~https://bitbucket.org/mobiflight/mobiflightfc/src/master/MSFS2020-module/mobiflight-event-module/modules/events.txt~~

oops, its actually moved to here: https://github.com/MobiFlight/MobiFlight-Connector/blob/main/MSFS2020-module/mobiflight-event-module/modules/events.txt

**Comment by JoergM-D** [comment]
Thanks a lot!
